src/rules/value-keyword-case/index.js

```diff
@@ -50,6 +57,10 @@ export default function (expectation) {
 
         if (parsedUnit !== false) { return }
 
+        const ignoreKeywords = options && options.ignoreKeywords || []
+
+        if (ignoreKeywords.indexOf(keyword) !== -1) { return }
```

Comment:
Like [`block-opening-brace-space-before`](https://github.com/stylelint/stylelint/blob/master/src/rules/block-opening-brace-space-before/index.js), can we use `matchesStringOrRegExp` here.